client/state/data-layer/wpcom/read/recommendations/sites/index.js

```diff
@@ -0,0 +1,67 @@
+
+/**
+ * External Dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	READER_RECOMMENDED_SITES_REQUEST
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { receiveRecommendedSites } from 'state/reader/recommended-sites/actions';
+
+export function requestRecommendedSites( { dispatch }, action, next ) {
+	const { seed = 1, number = 10, offset = 0 } = action.payload;
+	dispatch( http( {
+		method: 'GET',
+		path: '/read/recommendations/sites',
+		query: { number, offset, seed, posts_per_site: 0 },
+		apiVersion: '1.2',
+		onSuccess: action,
+		onFailure: action,
+	} ) );
+	next( action );
+}
+
+export const fromApi = response => {
```

Comment:
just curious but why `export const arrowFunction` instead of `export function fromApi` here? You used normal functions elsewhere... :D